flake.nix

```diff
@@ -54,6 +54,7 @@
           gcc
           rust
           postgresql
+          tesseract4
```

Comment:
What is this for?

Reply:
This is for OCRing the code from the issue screenshots.

Reply:
This is a one-off then, no need to force every developer shell to install this.

Reply:
I think it's useful. Let me put a Polly up for a couple hours and see.